core/src/main/java/org/dst/core/operatorset/DstSortedList.java

```diff
@@ -0,0 +1,21 @@
+package org.dst.core.operatorset;
+
+import org.dst.entity.SortedListEntity;
+
+import java.util.LinkedList;
+import java.util.List;
+
+public interface DstSortedList {
```

Comment:
Please add comments to describe this methods in interface.

Reply:
ok